src/Zoomba/APILibrary.py

```diff
@@ -275,24 +275,11 @@ def date_string_comparator(self, expected_date, actual_date, key, unmatched_keys
         if expected_date == actual_date:
             return
         else:
-            try:
-                expected_utc = datetime.datetime.strptime(expected_date, '%Y-%m-%dT%H:%M:%S')
-                actual_utc = datetime.datetime.strptime(actual_date, '%Y-%m-%dT%H:%M:%S')
+            expected_utc = _date_format(expected_date, key, unmatched_keys_list, "Expected")
+            actual_utc = _date_format(actual_date, key, unmatched_keys_list, "Actual")
+            if expected_utc and actual_utc:
                 self.date_comparator(expected_utc, actual_utc, key, unmatched_keys_list)
-            except ValueError:
```

Comment:
Why are we removing this whole block?  Was nothing valid?

Reply:
just abstraction to the _date_format() below